code/src/components/StartPage.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { useSelector } from "react-redux";
+import { IoIosSearch, IoMdBarcode } from 'react-icons/io'
+import styled from 'styled-components/macro'
+
+const Start = styled.section`
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+`
```

Comment:
This styling is found in many places of your code.
Maybe a good idea is to create a common component called Wrapper or similar that you can import everywhere you want to do this.

You could also then create styled components based on this one. Like:
```
const StartWrapper = styled(Wrapper)`
   some extra styling to that here...
`
```